squad/core/tasks/__init__.py

```diff
@@ -206,6 +206,8 @@ def __call__(test_run):
             issues[issue.test_name].append(issue)
 
         for test in test_parser()(test_run.tests_file):
+            if len(test['test_name']) > 256:
```

Comment:
please add TODO comment here so we know that this should be removed when the DB schema changes

Reply:
sure

Reply:
is there a reason to not increase the max size already?

whatever the new size is, we should document the limit explicitly. otherwise we can run into insanely long names again in the future.

Reply:
No particular reason. What's a good number? Also, do you think this may affect database performance? I have no clue on how postgres' data layout.

Reply:
I thought 256 was long enough... and than came CTS :(

Reply:
Java....

Reply:
> No particular reason. What's a good number? Also, do you think this may affect database performance? I have no clue on how postgres' data layout.

not sure. you would have to check. since tests is by far our largest table, any schema changes in it can be pretty disruptive as a migration that requires rewriting the underlying storage for the rows will take forever.

maybe we could go ahead with ignoring the tests with names that are too large for now. another option would be to hash these large names and then have another, small table that maps these hashes to the actual names.

Reply:
I agree with that. Running a simple SQL, I roughly got the amount of space used just to store test_names:

```
productionqareports=> SELECT SUM(length(name)) FROM core_test;
    sum     
------------
 9139882540
```

Since each character is 1 byte, the number above results in ~8GB.

Do you guys think it'd be a good idea to move all test names to an external table? I think so

Reply:
Just for the record there are 126552 unique test names in production: 

```
productionqareports=> SELECT COUNT(t.*) FROM (SELECT DISTINCT name FROM core_test ORDER BY name ASC) AS t;
 count  
--------
 126552
```

So given that qa-reports is over 200 million records in test table, that gives an average of 1500+ copies of each test name.